Service/InstallationService.php

```diff
@@ -6,7 +6,7 @@
 use App\Entity\Entity;
 use App\Entity\ObjectEntity;
 use CommonGateway\CoreBundle\Service\ComposerService;
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\Container;
```

Comment:
Are you sure about this?

Reply:
Nope die is daar onterecht blijven staan uit geklooi van gisteren avond